scripts/release/debian/deb_install.sh

```diff
@@ -27,7 +27,7 @@ function assert_consent {
     fi
 
     echo -n "$1 [Y/n] "
-    read consent
+    read -r consent
```

Comment:
https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-read

> -r
> If this option is given, backslash does not act as an escape character. The backslash is considered to be part of the line. In particular, a backslash-newline pair may not then be used as a line continuation.

The user should input `Y` or `n`. Specifying `-r` doesn't affect the functionality.

Reply:
https://www.shellcheck.net/wiki/SC2162 but I'll revert it